### PR TITLE
codacy fixes

### PR DESCRIPTION
**Pull request by archanabpatil**
Signed-off-by: archanabpatil <66412855+archanabpatil@users.noreply.github.com>

# Problem Statement
- Codacy fixes for couple of files

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: libs/s3/s3_multipart_test_lib.py [hunk 1]
```diff
@@ -225,10 +227,12 @@ def upload_precalculated_parts(self,
                         data, bucket_name, object_name, upload_id=upload_id,
                         part_number=int(partnum) + 1)
                     LOGGER.debug("Part : %s", str(part))
-                    uploaded_parts.append({"PartNumber": int(partnum) + 1, "ETag": part["ETag"]})
+                    uploaded_parts.append({"PartNumber": int(partnum) + 1,
+                                           "ETag": part["ETag"]}) # nosec
                     md5_digests[int(partnum)] = md5(data).digest()
```
Comment (apoorva-rao) [inline libs/s3/s3_multipart_test_lib.py:232]: It looks like the nosec comment needs to be added to line 232

**Review by apoorva-rao — CHANGES_REQUESTED** [review]
@archanabpatil please see inline comment for a minor change requested

File: libs/s3/s3_multipart_test_lib.py (hunk 1 above)
Comment (archanabpatil) [reply libs/s3/s3_multipart_test_lib.py:232]: sure

**Review by apoorva-rao — APPROVED** [review]
@archanabpatil please verify pytest collection and rebase PR if needed, before merging

**Review by apoorva-rao — APPROVED** [review]

**Review by VikasKumarC — APPROVED** [review]

**Pull request merged**